gitpod-io/gitpod
# Add invalid billing address notification

## Description


## Related Issue(s)

Fixes https://linear.app/gitpod/issue/EXP-1187

## How to test

Preview env: https://jp-semantic-marlin.preview.gitpod-dev.com/workspaces

1. Upgrade billing plan
2. Try using an invalid/valid address, check in stripe dashboard that taxes are calculated respectively
3. Update billing address to valid/invalid, check in stripe dashboard that taxes are calculated respectively 
4. If invalid billing address, check toast notification is shown

## Documentation


#### Preview status

gitpod:summary

## Build Options

<details>
<summary>Build</summary>

      Run the build with werft instead of GHA
      Run Leeway with `--dont-test`
</details>

<details>
<summary>Publish</summary>

</details>

<details>
<summary>Installer</summary>

  Add desired feature flags to the end of the line above, space separated
</details>

<details>
<summary>Preview Environment / Integration Tests</summary>

      If enabled this will build `install/preview`
      If enabled this will create the environment on GCE infra
      Saves cost. Untick this only if you're really sure you need a non-preemtible machine.
      Valid options are `all`, `workspace`, `webapp`, `ide`, `jetbrains`, `vscode`, `ssh`. If enabled, `with-preview` and `with-large-vm` will be enabled.
</details>

/hold
